assets/js/common/OperationModals/SaptuneSolutionOperationModal.jsx

```diff
@@ -9,7 +10,7 @@ const solutions = [
   {
     value: NOT_SELECTED,
     key: 'not_selected',
-    available: () => true,
+    available: (_hana, _app, currentlyApplied) => !currentlyApplied,
```

Comment:
I think we should add the usage of the 3rd variable in all `available` entries, even if it is a `_`.